workshops/fabric-e2e-rag/workshop.md

```diff
@@ -0,0 +1,606 @@
+---
+published: true
+type: workshop
+title: Building RAG (Retrieval augmented generation) Application on Microsoft Fabric & Azure Open AI
+short_title: Building RAG Application on Microsoft Fabric & Azure Open AI
+description: This workshop will guide you through the process of building a Retrieval augmented generation (RAG) application on Microsoft Fabric and Azure Open AI.
+level: beginner
+authors:
+  - Josh Ndemenge
+contacts:
+  - '@Jcardif'
+duration_minutes: 60
+tags: data, Microsoft Fabric, Azure Open AI
+banner_url: assets/architecture.png
+sections_title:
+  - Welcome
+  - Introduction
+  - Environment Setup
+  - Loading and Preprocessing PDF Documents
+  - Generating Embeddings and Storing them in a Vector Store
+  - Retrieving Relevant Documents and Answering Questions
+  - Conclusion
+
+wt_id: data-114676-jndemenge
+
+---
+
+# Welcome
+
+In this workshop, we'll demonstrate how to develop a context-aware question answering framework for any form of a document using [OpenAI models](https://azure.microsoft.com/products/ai-services/openai-service), [SynapseML](https://microsoft.github.io/SynapseML/) and [Azure AI Services](https://azure.microsoft.com/products/cognitive-services/). The source of data for this workshop is a PDF document, however, the same framework can be easily extended to other document formats too.
+
+## You'll learn how to: 
+
+- Pre-Process PDF Documents using [Azure AI Document Intelligence](https://azure.microsoft.com/products/ai-services/ai-document-intelligence) in Azure AI Services.
+- Perform text chunking using SynapseML.
+- Generate embeddings for the chunks using SynapseML and  [Azure OpenAI Services](https://azure.microsoft.com/products/cognitive-services/openai-service).
+- Store the embeddings in a vector store using [Azure Cognitive Search](https://azure.microsoft.com/products/search).
+- Build a question answering pipeline.
+
+
+## Pre-requisites
+
+
+| | |
+|----------------------|------------------------------------------------------|
+| Azure account        | [Get a free Azure account](https://azure.microsoft.com/free) |
```

Comment:
Are free accounts definitely able to deploy these resources? We found that we had to make a lot of modifications for our OpenAI+Search sample to be compatible with free accounts.